layouts/shortcodes/pentest-report-requirements.html

```diff
@@ -1,3 +1,3 @@
-If you want a pentest report, you generally must set up a test of at least two (2) credits. 
-If you have a one (1) credit pentest, you'll still have access to the non-report
+If you want a pentest report, you generally must set up a test of at least eight (8) credits. 
```

Comment:
Probably @yogipetkar or @TheInstaGrahame highlighted an error here in some internal chat. I think I've fixed this now, and await your approval.